html-forms/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<body>
+  Email
+  <form action="none">
+    <input type="text" name="WhyCoding" id="WhyIsCodingCool">
+    <label for="WhyIsCodingCool"><div>Why is coding cool?</div></label>
```

Comment:
The value of a `for` attribute must match the value of the `id` attribute of the form control element this is labelling